etc/deploy/gcp/gcp-doco-script.sh

```diff
@@ -106,7 +110,17 @@ gcloud iam service-accounts add-iam-policy-binding ${SERVICE_ACCOUNT} \
     --role roles/iam.workloadIdentityUser \
     --member "${CLOUDSQLAUTHPROXY_WI}"
 
-gcloud sql databases create pachyderm -i ${CLOUDSQL_INSTANCE_NAME}
+gcloud iam service-accounts create ${LOKI_GSA_NAME}
+
+gcloud projects add-iam-policy-binding ${PROJECT_ID} \
+    --member="serviceAccount:${LOKI_SERVICE_ACCOUNT}" \
+    --role="${ROLE2}"
+
+gcloud iam service-accounts keys create "${LOKI_GSA_NAME}-key.json" --iam-account="$LOKI_SERVICE_ACCOUNT"
+
+kubectl create secret generic loki-service-account --from-file="${LOKI_GSA_NAME}-key.json"
```

Comment:
What exactly is this secret protecting? Is this how we give pachd object storage permissions, too, or do we somehow derive it elsewhere?

Reply:
It is used to give the Loki pod permission to write to object storage:

```yaml
loki-stack:
  loki:
    env:
    - name: GOOGLE_APPLICATION_CREDENTIALS
      value: /etc/secrets/${LOKI_GSA_NAME}-key.json
    extraVolumes:
      - name: loki-service-account
        secret:
          secretName: loki-service-account
    extraVolumeMounts:
      - name: loki-service-account
        mountPath: /etc/secrets
```